### PR TITLE
livedst: Add openapi spec for the API

### PR DESCRIPTION
**Pull request by greglucas**
Adding the openapi spec for the livedst models

### CODE REVIEW

File: dst/livedst-openapi.json [hunk 1]
```diff
@@ -0,0 +1,297 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "LiveDst API",
+    "description": "The LiveDst API enables easier public access to the live Dst predictions made\nby CU's space weather machine learning group.\n\nSome useful links:\n- [LiveDst plots](https://swx-trec.com/dst)\n- [The SWx TREC Model Staging Platform](https://swx-trec.com)",
```
Comment (jennyknuth) [inline dst/livedst-openapi.json]: Let's try to consistently brand it as "Live Dst".

File: dst/livedst-openapi.json [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+      }
+    },
+    "/livedst_model": {
+      "get": {
+        "tags": [
+          "dst"
+        ],
+        "summary": "Get predictions grouped by model-time",
+        "description": "Group model results by the model run time (model0, model1, ..., modelN)",
+        "operationId": "findPetsByTags",
```
Comment (jennyknuth) [inline dst/livedst-openapi.json]: no pets here!

File: dst/livedst-openapi.json (hunk 1 above)
Comment (jennyknuth) [reply dst/livedst-openapi.json]: Maybe "The Live Dst API enables easier public access to the real-time Dst…"

**Review by jennyknuth — CHANGES_REQUESTED** [review]

File: dst/livedst-openapi.json [hunk 3]
```diff
@@ -0,0 +1,297 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "LiveDst API",
```
Comment (jennyknuth) [inline dst/livedst-openapi.json]: Live Dst API

File: dst/livedst-openapi.json (hunk 1 above)
Comment (jennyknuth) [reply dst/livedst-openapi.json]: "Live Dst plots"

Comment (jennyknuth) [reply dst/livedst-openapi.json]: And maybe the link for swx-trec.com should say "Related tools and applications" or something. A platform sounds intimidating and technical!

File: dst/livedst-openapi.json (hunk 2 above)
Comment (greglucas) [reply dst/livedst-openapi.json]: Thanks!

File: dst/livedst-openapi.json (hunk 1 above)
Comment (greglucas) [reply dst/livedst-openapi.json]: Made all those changes. I thought it was all one word, so thanks for pointing that out!

**Review by jennyknuth — APPROVED** [review]

File: dst/livedst-openapi.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
   "openapi": "3.0.3",
   "info": {
-    "title": "LiveDst API",
-    "description": "The LiveDst API enables easier public access to the live Dst predictions made\nby CU's space weather machine learning group.\n\nSome useful links:\n- [LiveDst plots](https://swx-trec.com/dst)\n- [The SWx TREC Model Staging Platform](https://swx-trec.com)",
+    "title": "Live Dst API",
+    "description": "The Live Dst API enables easier public access to the live Dst predictions made\nby CU's space weather machine learning group.\n\nSome useful links:\n- [Live Dst plots](https://swx-trec.com/dst)\n- [Related space weather tools and applications](https://swx-trec.com)",
```
Comment (jennyknuth) [inline dst/livedst-openapi.json]: last tiny thing, do you want the lowercase "live Dst predictions" to be changed to "real time Dst predictions" so as not to be confusing, or do you like it used again as an adjective?

Comment (greglucas) [reply dst/livedst-openapi.json]: Good suggestion. I just made that change too.

**Pull request merged**